pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -420,10 +422,33 @@ def get_treeinfo_data(remote, remote_url):
 
         return treeinfo_serialized
 
+    def get_sync_details(remote, url, sync_policy, version):
+        with tempfile.TemporaryDirectory("."):
+            try:
+                result = get_repomd_file(remote, url)
+                repomd_path = result.path
+                repomd = cr.Repomd(repomd_path)
+                repomd_checksum = get_sha256(repomd_path)
+            except Exception:
+                repomd_checksum = None
+
+        return {
+            "url": remote.url,
```

Comment:
Note that this is explicitly the remote url, not the "actual" url, because presumably we want to optimize even if a different mirror is used from a mirrorlist.

Reply:
yes i did notice that :+1:

Reply:
ha, brain :brain: